db/version_set.cc

```diff
@@ -1758,6 +1762,35 @@ void VersionStorageInfo::ComputeExpiredTtlFiles(
   }
 }
 
+void VersionStorageInfo::ComputeFilesMarkedForPeriodicCompaction(
+    const ImmutableCFOptions& ioptions,
+    const uint64_t periodic_compaction_time) {
+  assert(periodic_compaction_time > 0);
+
+  files_marked_for_periodic_compaction_.clear();
+
+  int64_t _current_time;
```

Comment:
I remember we decided that this leading "_" can be used in arguments of constructor. I don't remember we decided that we can use it for normal variables.

Reply:
changed.